UACloudLibraryServer/Models/UANameSpace.cs

```diff
@@ -83,15 +83,12 @@ public class UANameSpaceMetadata
         /// </summary>
         public string[] SupportedLocales { get; set; }
 
-        public uint NumberOfDownloads { get; set; }
```

Comment:
Do we want to keep returning the number of downloads? I think this is a good indication of popularity.